### PR TITLE
Added loading display functionality

### LINKED ISSUE #27: Add loading indicator
As an impatient user, I want to see some indication that data is loading.
Solution: add loading gif when functions are getting data from api's.

### PR DESCRIPTION
**Pull request by carlthedev**
Relates #27

The simplest way i could think of showing the loading gif is to toggle it with display none and flex. I put this in the relevant functions before any results are rendered on the page with a set timeout of 2 seconds, so that we see the loading gif for a minimum of 2 seconds everytime.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -41,6 +41,19 @@ body {
...
+
+#gif-load {
+  display: none;
+  flex-direction: column;
+  align-items: center;
+}
+
+#gif-load img {
+  width: auto;
+  height: 20vw;
```
Comment (camelPhonso) [inline styles.css]: could you make this 10vw? I think it makes a big difference, particularly when we switch to bigger screens

File: main.js [hunk 2]
```diff
@@ -217,15 +225,22 @@ const submitBtn = document.querySelector("#submit-button");
 submitBtn.addEventListener("click", (e) => {
   e.preventDefault();
   if (document.getElementById('city-input').value.length !== 0) {
-    startSearch();
+    // show the loading gif image
+    gifFig.style.display = "flex";
```
Comment (camelPhonso) [inline main.js:231]: we should add a line on 228 with canvas.InnerHTML = '' to remove the Spot Check logo while the loading animation displays.

**Review by camelPhonso — CHANGES_REQUESTED** [review]
Looks great dude, really love the simple solution 🎉 

I added two comments for basic changes I think would make the usage a bit nicer, but other than those minor tweaks I think we're good

File: styles.css (hunk 1 above)
Comment (carlthedev) [reply styles.css]: reply test

Comment (carlthedev) [reply styles.css]: changed now

**Review by camelPhonso — APPROVED** [review]

**Pull request merged**